RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -244,15 +244,15 @@ public ObservableCollection<CodeExplorerItemViewModel> Projects
             }
         }
 
-        private void ParserState_StateChanged(object sender, EventArgs e)
+        private void ParserState_StateChanged(object sender, ParserStateEventArgs e)
         {
             if (Projects == null)
             {
                 Projects = new ObservableCollection<CodeExplorerItemViewModel>();
             }
 
             IsBusy = _state.Status < ParserState.ResolvedDeclarations;
-            if (_state.Status != ParserState.ResolvedDeclarations)
+            if (e.State != ParserState.ResolvedDeclarations)
```

Comment:
That is not an issue--I do not calculate this off the module states, but manually fire it.